### DISCUSSION TITLE
File size limit

### DISCUSSION THREAD

**Discussion by Gonched** [TumblThreeApp/TumblThree#327] [Q&A (Question & Answer)]
Would it be possible to get a limit on the size of the files being downloaded?  Some idiot is uploading 4 gig videos to a blog I download and is putting a jpg extension on them.  Thanks.

**Comment by thomas694** [comment]
As far as I know there aren't blogs where everyone can post to. So the blog owner is either sharing his own content or reblogs. Depending on how it manifests exactly and how important the blog is to you, you could among other things either message the owner and tell him there's something wrong with the posts (unintentionally wrong extension by e.g. defect tool), report the blog if malicious content (intentionally wasting bandwidth) posted or cross the blog off from your list.

**Comment by thomas694** [comment]
The issue has been closed. You can still comment. Feel free to ask for reopening the issue if needed.
